.github/workflows/pull-request-size.yml

```diff
@@ -0,0 +1,17 @@
+name: "Pull Request Size Labeler"
+
+on: [pull_request]
+
+jobs:
+  labeler:
+    runs-on: ubuntu-latest
+    name: Label the PR size
+    steps:
+      - uses: codelytv/pr-size-labeler@v1
```

Comment:
Upstream enhancement to allow specifying the size labels is still pending: https://github.com/CodelyTV/pr-size-labeler/pull/26

Up to you folks whether this is worth skipping this migration for now (I will ensure there is a solution before decommissioning) or if it is worth renaming the repository `size/*` labels with lowercase to match this action after merging this.

Reply:
we have no preference, so i'll rename the labels to lowercase. thanks!